core/game_object.lua

```diff
@@ -2419,165 +2423,237 @@ function loadAPIs()
     })
 
     -------------------------------------------------------------------------------------------------
-    ----- API CODE GameObject.Palette
+    ----- API CODE GameObject.AltTexture
     -------------------------------------------------------------------------------------------------
 
-    SMODS.local_palettes = {}
-    SMODS.Palettes = { Types = {} }
-    SMODS.Palette = SMODS.GameObject:extend {
-        obj_table = SMODS.local_palettes,
+    SMODS.alt_textures = {}
+
+    SMODS.AltTextures = {
+        Types = {"Suit", "Tarot", "Planet", "Spectral", "Joker", "Enhanced", "Back", "Seal", "Voucher", "Booster", "Tag", "Blind"},
```

Comment:
The organization here is a bit weird, Types is not a 'real' element of the table